x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/job_details/extract_job_details.js

```diff
@@ -139,6 +147,14 @@ export function extractJobDetails(job) {
     items: filterObjects(job.model_size_stats).map(formatValues)
   };
 
+  const datafeedTimingStats = {
```

Comment:
Does this part also need a check if `timing_stats` is available at all?

Reply:
hmm, I actually don't think the check is needed at all even on the lines above.
`timing_stats` is always there, even when the datafeed hasn't been run.